app/routes/goal_route.py

```diff
@@ -0,0 +1,101 @@
+from flask import Blueprint, jsonify, request
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle